src/Calculator.UWP/Calculator.UWP.csproj

```diff
@@ -57,7 +57,7 @@
     <UseVSHostingProcess>false</UseVSHostingProcess>
     <ErrorReport>prompt</ErrorReport>
     <Prefer32Bit>true</Prefer32Bit>
-    <UseDotNetNativeToolchain>true</UseDotNetNativeToolchain>
+    <UseDotNetNativeToolchain>false</UseDotNetNativeToolchain>
```

Comment:
Is this change wanted?

Reply:
Yes, not using the .NET native output of the build, so it make the build longer for no useful reason.